### PR TITLE
[MXNET-1402] MXNet docs change for 1.4.1 release

### PR DESCRIPTION
**Pull request by junrushao1994**

### CODE REVIEW

File: docs/install/index.md [hunk 1]
```diff
@@ -39,10 +39,11 @@
 Indicate your preferred configuration. Then, follow the customized commands to install MXNet.
 
 <div class="dropdown">
-  <button class="btn current-version btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.4.0
+  <button class="btn current-version btn-primary dropdown-toggle" type="button" data-toggle="dropdown">v1.4.1
   <span class="caret"></span></button>
   <ul class="dropdown-menu opt-group">
-    <li class="opt active versions"><a href="#">v1.4.0</a></li>
+    <li class="opt active versions"><a href="#">v1.4.1</a></li>
+    <li class="opt versions"><a href="#">v1.4.0</a></li>
```
Comment (TaoLv) [inline docs/install/index.md]: Do we still need to provide option for 1.4.0? Previously we kept option for the last patch release of a minor release.

Comment (junrushao1994) [reply docs/install/index.md]: Good catch! I was not aware of it before. @szha Could you advise?

Comment (szha) [reply docs/install/index.md]: Let's stick to the convention and provide the latest patch release of each minor release.

File: docs/_static/mxnet-theme/index.html [hunk 2]
```diff
@@ -23,9 +23,9 @@
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>MXNet 1.4.0 Released</h3>
+        <h3>MXNet 1.4.1 Released</h3>
         <p>This release introduces the Java Inference API and Julia API, as well as Control Flow Operators, MKLDNN optimizations, and SVRG optimization.</p>
```
Comment (aaronmarkham) [inline docs/_static/mxnet-theme/index.html]:
```suggestion
        <p>This patch release features bug fixes and performance improvements.</p>
```

File: docs/install/index.md [hunk 3]
```diff
@@ -190,7 +197,7 @@ $ pip install mxnet --pre
 MXNet offers MKL pip packages that will be much faster when running on Intel hardware.
 Check the chart below for other options, refer to <a href="https://pypi.org/project/mxnet/">PyPI for other MXNet pip packages</a>, or <a href="validate_mxnet.html">validate your MXNet installation</a>.
 
-<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages-1.4.0.png" alt="pip packages"/>
+<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages-1.4.1.png" alt="pip packages"/>
```
Comment (aaronmarkham) [inline docs/install/index.md]: I updated this image.
```suggestion
<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png" alt="pip packages"/>
```

File: docs/install/index.md [hunk 4]
```diff
@@ -324,7 +331,7 @@ $ pip install mxnet-cu92 --pre
...
```
Comment (aaronmarkham) [inline docs/install/index.md]: I updated this image.
```suggestion
<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png" alt="pip packages"/>
```

File: docs/install/index.md [hunk 5]
```diff
@@ -640,7 +647,7 @@ $ pip install mxnet --pre
...
```
Comment (aaronmarkham) [inline docs/install/index.md]: I updated this image.
```suggestion
<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png" alt="pip packages"/>
```

File: docs/install/index.md [hunk 6]
```diff
@@ -928,7 +935,7 @@ $ pip install mxnet --pre
...
```
Comment (aaronmarkham) [inline docs/install/index.md]: I updated this image.
```suggestion
<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png" alt="pip packages"/>
```

File: docs/install/index.md [hunk 7]
```diff
@@ -1059,7 +1066,7 @@ $ pip install mxnet-cu92 --pre
...
```
Comment (aaronmarkham) [inline docs/install/index.md]: I updated this image.
```suggestion
<img src="https://raw.githubusercontent.com/dmlc/web-data/master/mxnet/install/pip-packages.png" alt="pip packages"/>
```

**Comment by szha** [comment]
@aaronmarkham @junrushao1994 what's the status? We already released it in github for some time so we should try to get this in and make announcement soon

**Comment by junrushao1994** [comment]
@szha We are waiting for @zachgk for the Scala build, and other things should be complete

**Comment by pinaraws** [comment]
@mxnet-label-bot add[Doc, pr-awaiting-response]

**Comment by aaronmarkham** [comment]
Hi @junrushao1994 - Doesn't look like step 6 was done:(https://cwiki.apache.org/confluence/display/MXNET/Release+Process#ReleaseProcess-3.2.UpdatetheMXNetwebsiteupdate-mxnet-website)
Are we ready for that, or still waiting on a different step to finish?

**Comment by junrushao1994** [comment]
Hey @aaronmarkham, I haven't had CI access yet. Could you help with this? Thanks!

**Comment by aaronmarkham** [comment]
I updated CI... there's a run in progress, so we should see an update around EOD.

**Pull request merged**